app/services/reporting/agency_and_sp_report.rb

```diff
@@ -67,11 +67,12 @@ def service_providers
     end
 
     def facial_match_issuers
-      @facial_match_issuers ||= Profile.where(active: true).where(
-        'verified_at <= ?',
-        report_date.end_of_day,
-      ).where(idv_level: Profile::FACIAL_MATCH_IDV_LEVELS).
-        pluck(:initiating_service_provider_issuer).uniq
+      @facial_match_issuers ||= Reports::BaseReport.transaction_with_timeout do
+        Profile.active.verified.facial_match_opt_in.
+          where('verified_at <= ?', report_date.end_of_day).
```

Comment:
Is the `verified` scope on the prior line redundant with this?

Reply:
yeah, it can't be active without verified so idk why I am the way that I am.